code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -80,7 +80,6 @@
 	name = "cable restraints"
 	desc = "Looks like some cables tied together. Could be used to tie something up."
 	icon_state = "cuff_red"
-	item_state = "coil_red"

```

Comment:
Probably be safer to keep the icon just incase someone does something dumb and spawns this down the line